index.html

```diff
@@ -0,0 +1,175 @@
+                                <li class="html"><a href="" class="html">HTML</a></li>
+                                <li class="css"><a href="" class="css">CSS</a></li>
+                                <li class="javaScript"><a href="" class="javascript">JavaScript</a></li>
+                            </ul>
+                        <button type="button" class="more-projects">See project</button>
+                    </div>
+                </div>  
+    </section>
+
+    <!-- About myself section -->
+    <!-- <section class="about_myself">
+        <h2>About Myself</h2>
+            <p>Hello I'm a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+<a class="connection" href="#">LET'S CONNECT</a>
+<ul class="connect-icon-container">
+    <li class="twitter-icon"><a href=""><img src="Icons/twitter.png" alt="twitter"></a></li>
+    <li class="linkedin-icon"><a href=""><img src="Icons/linkedin.png" alt="linkedin"></a></li>
+    <li class="medium-icon"><a href=""><img src="Icons/medium.png" alt="medium"></a></li>
+    <li class="github-icon"><a href=""><img src="Icons/github.png" alt="github"></a></li>
+    <li class="peace-icon"><a href=""><img src="Icons/peace.png" alt="peace"></a></li>
+    </ul>
+    <button type="button" class="resume">Get my resume</button>
+    <ul class="categories">
+        <li class="category-language"><span><h3>Languages</h3></span><button class="modal-view-arrow" type="button"><img src="./Icons/down-arrow.png" alt="down-arrow"></li></li>
+        <div class="modal-view-container">
+            <div class="modal-view">
+                <div class="modal-view-icon">  
+                    <img src="./Icons/javascript-Icon.png" alt="javaScript-icon">
+                </div>
+                    <h3>JavaScript</h3>
+                </div>
+                <div class="modal-view">
+                    <div class="modal-view-icon">
+                        <img src="./Icons/HTML-Icon.png" alt="HTML-icon">
+                    </div>
+                    <h3>HTML</h3>
+                </div>
+                <div class="modal-view">
+                    <div class="modal-view-icon">
+                        <img src="./Icons/CSS-Icon.png" alt="CSS-icon">
+                </div>
+                <h3>CSS</h3>
+                    </div>
+            </div>
+        <li><span><h3>Frameworks</h3></span><button class="modal-view-arrow" type="button"><img src="./Icons/left-arrow.png" alt="left-arrow"></button></li>
+        <img src="./Icons/separator.png" alt="bottom-separator">
+        <li><span><h3>Skills</h3></span><button class="modal-view-arrow" type="button"><img src="./Icons/left-arrow.png" alt="left-arrow"></button></li>
+        <img src="./Icons/separator.png" alt="bottom-separator">
+    </ul>
+</section>
+    </section> --> -->
```

Comment:
- [ ] Please do not push commented out code to your code base, They will make your project look messy. Kindly remove them from your code base